src/playlist-loader.js

```diff
@@ -33,6 +33,30 @@ export const forEachMediaGroup = (master, callback) => {
   });
 };
 
+/**
+ * Checks whether xhr request was redirected and returns correct url depending
+ * on `handleManifestRedirects` option
+ *
+ * @api private
+ *
+ * @param  {String} url - an url being requested
+ * @param  {XMLHttpRequest} req - xhr request result
+ *
+ * @return {String}
+ */
+export const resolveManifestRedirect = (handleManifestRedirect, url, req) => {
```

Comment:
maybe this makes more sense to be in the `resolve-url.js` file now that it's used for HLS and DASH?